gradle.properties

```diff
@@ -10,9 +10,9 @@ puppycrawlCheckstyleVersion=8.18
 testngVersion=7.4.0
 gsonVersion=2.8.8
 
-stdlibCryptoVersion=2.2.1
-stdlibLogVersion=2.2.1
-stdlibTimeVersion=2.2.1
-stdlibIoVersion=1.2.1
-observeVersion=1.0.2
-observeInternalVersion=1.0.2
+stdlibCryptoVersion=2.2.2-20220510-101700-bd6a537
+stdlibLogVersion=2.3.0-20220510-164700-f61e84f
```

Comment:
Since we are updating the log version to a minor, we need to update the module version also to a minor one.

Reply:
Updated